artview/components/display.py

```diff
@@ -0,0 +1,297 @@
+"""
+display.py
```

Comment:
what is the purpose of this file? the name is to general that may mislead a code reader.